ui/actions/providers/providers.ts

```diff
@@ -260,6 +278,7 @@ export const updateCredentialsProvider = async (
   const providerType = formData.get("providerType");
```

Comment:
```suggestion
  const providerType = formData.get("providerType") as ProviderType;
```

As a quick suggestion, we can cast this to be safer, but the end solution is to create a Zod schema for these forms. Also, we should use useFormState for these server actions, form-related.

Reply:
Agreed. Like we said yesterday, that’s already planned. We’ll take care of it when the time comes.